android/app/src/main/java/org/electroncash/electroncash3/ColdLoad.kt

```diff
@@ -143,13 +130,17 @@ class ColdLoadDialog : AlertDialogFragment() {
      */
     private fun updateStatusText(tx: PyObject) {
         try {
-            val txInfo = daemonModel.wallet!!.callAttr("get_tx_info", tx)
-
-            // Check if the transaction can be processed by this wallet or not
-            if (txInfo["amount"] == null) {
-                idTxStatus.setText(R.string.transaction_unrelated)
+            if (etTransaction.text.isBlank()) {
+                idTxStatus.setText(R.string.empty)
             } else {
-                idTxStatus.setText(txInfo["status"].toString())
+                // Check if the transaction can be processed by this wallet or not
+                val txInfo = daemonModel.wallet!!.callAttr("get_tx_info", tx)
+
+                if (txInfo["amount"] == null && txInfo["status"].toString() != "Signed") {
```

Comment:
The string comparison isn't safe because the status string should be translated into other languages. Though this translation apparently doesn't work at the moment, which a bug that I'll look into myself.

Anyway, it's best not to rely on the wording of UI strings from the back-end, because they might change in the future. So I suggest we just call `canBroadcast` instead.

Reply:
> this translation apparently doesn't work at the moment, which a bug that I'll look into myself.

I've just pushed a fix for this.